app/validators/questions/question_validator.py

```diff
@@ -2,9 +2,36 @@
 
 
 class QuestionValidator(Validator):
+    ANSWER_LABEL_MISSING_MULTIPLE_ANSWERS = (
+        "Answer label must be provided for questions with multiple answers"
+    )
     question = {}
 
     def __init__(self, schema_element):
         super().__init__(schema_element)
         self.question = schema_element
+        self.answers = self.question.get("answers", [])
         self.context["question_id"] = schema_element["id"]
+
+    def validate(self):
+        if self.question["type"] != "MutuallyExclusive":
+            self._validate_answer_labels()
+
+        return self.errors
+
+    def _validate_answer_labels(self):
+        if len(self.answers) < 2:
+            return None
+
+        for answer in self.answers:
+            if not answer.get("label"):
+                if (
+                    answer["type"] == "Checkbox"
+                    and (answer == self.answers[-1])
+                    and len(answer["options"]) < 2
+                ):
```

Comment:
I think a single checkbox should only be used like this when there are two answers, although this doesn't seem to be documented at all in the design system.

This also enforces a label for the first answer, which works for the current schemas as the first answer is a `Number`. This pattern may need to be used with other answer types e.g. Radio, so I think we should allow the first answer to not have a label.